plugins/board-resources/package.json

```diff
@@ -12,39 +12,42 @@
     "format": "prettier --write --plugin-search-dir=. src && eslint --fix src"
   },
   "devDependencies": {
-    "svelte-loader": "^3.1.2",
```

Comment:
Could you please try rush apply-templates to format packages.json files.

Reply:
I have run it, but I do not see any changes in package.json

Reply:
ok